packages/chatgpt/src/Adaptor.js

```diff
@@ -78,12 +86,60 @@ export function prompt(message, opts) {
 
     const payload = {
       model: 'o3-mini-2025-01-31',
-      reasoning_effort: 'low',
       messages: [{ role: 'user', content: resolvedMessage }],
       ...resolvedOpts,
     };
 
     const msg = await client.chat.completions.create(payload);
+    console.log('√ Prompt operation completed');
+    return composeNextState(state, msg);
+  };
+}
+
+/**
+ * Prompt GPT deep research interface
+ * @example
+ * deepResearch(
+ *   `Verify if this healthcare facility is legitimate:
```

Comment:
Is this example too close to your current project? We shouldn't be making client project logic public.